launcher/launcher/main.go

```diff
@@ -152,7 +125,7 @@ func getExitCode(isHardened bool, restartPolicy spec.RestartPolicy, err error) i
 	return exitCode
 }
 
-func startLauncher() error {
+func startLauncher(launchSpec spec.LaunchSpec) error {
```

Comment:
Can serialConsole pass in here like the launchSpce here instead of a global variable?